### PR TITLE
#23395 [CUBRID] Extend navigator to display db_server information

### PR DESCRIPTION
**Pull request by longhaseng52**
- Extend CUBRID plugin to display 'db_server' information on the navigator
- The db_server object in the CUBRID catalog is connection information for a remote database server supported by CUBRID 11.2 or later (database link)

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @longhaseng52 

Thanks for your contribution.

We will look at it.

**Comment by LonwoLonwo** [comment]
Please, all next commits in this branch start from the ticket number. like `#23395 fix code style`

File: plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridServer.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    @NotNull
+    @Property(viewable = true, order = 7)
+    public String getOwner() {
+        return owner;
+    }
+
+    @Nullable
+    @Override
+    @Property(viewable = true, order = 8)
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridServer.java]:
```suggestion
    @Property(viewable = true, length = PropertyLength.MULTILINE, order = 8)

```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridServer.java]: Please add a multiline parameter.

**Review by LonwoLonwo — APPROVED** [review]

**Comment by E1izabeth** [comment]
Hi, @longhaseng52 
This PR cannot be built due to the following error:
```
Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:4.0.4:compile (default-compile) on project org.jkiss.dbeaver.ext.cubrid: Compilation failure: Compilation failure: 
[ERROR] /var/jenkins_home/jobs/DBeaverDesktop/jobs/dbeaver-ce/workspace/dbeaver/plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridServer.java:[95] 
[ERROR] 	@Property(viewable = true, length = PropertyLength.MULTILINE, order = 8)
[ERROR] 	                                    ^^^^^^^^^^^^^^
[ERROR] PropertyLength cannot be resolved to a variable
[ERROR] 1 problem (1 error)
```
Could you fix it?

**Comment by longhaseng52** [comment]
Hello @E1izabeth,

I got it fixed now.

**Comment by E1izabeth** [comment]
Thank you

**Review by E1izabeth — APPROVED** [review]

**Pull request merged**